common/src/main/java/org/jboss/pnc/common/monitor/PullingMonitor.java

```diff
@@ -67,22 +67,20 @@
 
     public PullingMonitor() {
```

Comment:
I know, it is not part of this PR, but shouldn't it be called PollingMonitor?

Reply:
This can be renamed in another PR :)